src/current/v23.1/sso-sql.md

```diff
@@ -1,158 +1,257 @@
+    {% include_cached copy-clipboard.html %}
+    ~~~shell
+    SET CLUSTER SETTING server.jwt_authentication.issuers = 'https://accounts.google.com';
 
-	{{site.data.alerts.callout_danger}}
-	Many third-party token issuers, including GCP and Azure, will by default create tokens with a generic default audience. It is best practice to limit the scope of access tokens as much as possible, so if possible, we recommend issuing tokens with only the required audience value corresponding to the `audience` configured on the cluster.
+    ~~~
 
-	By extension, if your provider allows you to specify scopes or permissions on the token, you should specify these as restrictively as possible, while still allowing for the functions intended for the service account or user.
-	{{site.data.alerts.end}}
+1. `server.jwt_authentication.audience`
 
-## Configure your cluster's identity mapping
+    The ID of your cluster as specified by the IdP, or a JSON array of such names. One of the audience values here must match the `audience` claim of an access token, or it will be rejected.
 
-### `server.identity_map.configuration`
+    {{site.data.alerts.callout_danger}}
+    Many third-party token issuers, including GCP and Azure, will by default create tokens with a generic default audience. It is best practice to limit the scope of access tokens as much as possible, so if possible, we recommend issuing tokens with only the required audience value corresponding to the `audience` configured on the cluster.
 
-Format: `<external issuer> <external user ID> <SQL username>`
+    By extension, if your provider allows you to specify scopes or permissions on the token, you should specify these as restrictively as possible, while still allowing for the functions intended for the service account or user.
+    {{site.data.alerts.end}}
 
-Specifies mapping of subject names to SQL usernames, for each allowed IdP. For example, a configuration of:
+    {% include_cached copy-clipboard.html %}
+    ~~~shell
+    SET CLUSTER SETTING server.jwt_authentication.audience = '984901724939-njig7lkv7k724rbv2hllvr4of8ul7th7.apps.googleusercontent.com';
+    ~~~
 
-`https://accounts.google.com   /^([9-0]*)$   gcp_\1`
+1. `server.jwt_authentication.claim`
 
-would yield a mapping where the SQL username for each GCP-provisioned service account would be `gcp_{user ID}`, e.g. `gcp_1234567` for a service account with ID `1234567`.
+    The value of `email` indicates the email field in the JWT.
 
-## Authenticate to your cluster with your JWT token
+    {% include_cached copy-clipboard.html %}
+    ~~~shell
+    SET CLUSTER SETTING server.jwt_authentication.claim = 'email';
+    ~~~
 
-To provision SQL cluster access for application users or service accounts, you must provision JWT tokens. There are many ways to do this, which are beyond the scope of this tutorial.
+1. `server.jwt_authentication.jwks`
```

Comment:
Is this largely a duplicate of above?

Reply:
yes, thanks, tried to consolidate things.